cost/github/available_seats/available_seats.pt

```diff
@@ -0,0 +1,105 @@
+name "GitHub.com Available Seats Report"
+rs_pt_ver 20180301
+short_description "Gets the number of available seats for a licensed GitHub.com Org and creates an incident if they are out of the policy range."
```

Comment:
The short description should have a link to the README file. Checkout another policy for an example.

Reply:
Example: https://github.com/rightscale/policy_templates/blob/master/cost/volumes/unattached_volumes/uav_policy.pt#L4

Reply:
Fixed in 117facc